src/assets/doc_output.json

```diff
@@ -0,0 +1,129 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "version": "0.0.1-alpha.0",
+    "title": "ENS Metadata Service",
+    "description": "Set of endpoints to query ENS metadata and more",
+    "x-logo": {
+      "url": "/assets/logo.svg",
+      "altText": "Ethereum Name Service"
+    }
+  },
+  "host": "http://localhost:8080",
+  "basePath": "/",
+  "tags": [],
+  "schemes": [
+    "http"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/{contractAddress(0x[a-fA-F0-9]{40})}/{tokenId}": {
+      "get": {
+        "tags": [],
+        "description": "ENS NFT metadata",
+        "parameters": [
+          {
+            "name": "contractAddress",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Contract Address of your stored NFT"
+          },
+          {
+            "name": "tokenId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Namehash(v1) /Labelhash(v2) of your ENS name"
```

Comment:
Can you set a link to https://docs.ens.domains/contract-api-reference/name-processing#hashing-names